graph/config/cytoscape/cytoscape.go

```diff
@@ -108,6 +108,7 @@ type NodeData struct {
 	IsOutside             bool                `json:"isOutside,omitempty"`             // true | false
 	IsRoot                bool                `json:"isRoot,omitempty"`                // true | false
 	IsServiceEntry        *graph.SEInfo       `json:"isServiceEntry,omitempty"`        // set static service entry information
+	HasWorkloadEntry      bool                `json:"hasWorkloadEntry,omitempty"`      // true (this node has a corresponding WorkloadEntry) | false
```

Comment:
It would probably be useful to pass back some of the info from the WE here but I'm unsure exactly what info is useful or how the front end would use it so I left it a boolean for now. I can instead update this to be an empty struct for now that way it'd be easy to add that info later and the front end could remain unchanged? e.g.
```
struct WEInfo {}
```
```
if (node.hasWorkloadEntry !== undefined) { ... }
```

Reply:
No, if there isn't anything we can think of then just leave it as a boolean.  No need to over-engineer.